### PR TITLE
Fix toc and bibliography issue - compatibility with Jupyter-Book upgrade

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* The Jupyter-book upgrade was made on The Turing Way a few days ago which created some incompatibility with the _toc and bibliography files
 
### List of changes proposed in this PR (pull-request)

* [x] Fix toc file structure
* [x] Fix other minor issues such as label and missing files from toc
* [ ] Fix bibliography issue 
   * replaces {cite} with {cite:t} as described in JB handbook: https://jupyterbook.org/content/citations.html?highlight=biblio
   * I can't figure what the issue is. Here is the error that I get in my local build
```
/Users/msharan/github/the-turing-way/book/website/afterword/bibliography.md:6: WARNING: Unknown directive type "bibliography".
...
```
Followed by "WARNING: Unknown interpreted text role "cite:t"" for all the citation made in the book

### What should a reviewer concentrate their feedback on?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@martinagvilas @sgibson91 can you please help troubleshoot the bibliography issue?

**Comment by malvikasharan** [comment]
In this PR I have fixed the errors generated in local builds, one was that 'ci-travis.md' was not included in _toc. @BatoolMM did you intentionally leave out Travis chapter from the book? If yes, then I can remove that completely.

**Comment by martinagvilas** [comment]
@malvikasharan I'll take a look

**Comment by BatoolMM** [comment]
Yes, both Kristie and @bsipocz recommended that we remove the Travis part completely. It's a mystery to me how it made a comeback in this PR. I removed it again from the `_toc` and made some cleaning for the glossary and references within the IC chapter. Thank you so much for adding the labels, which we completely forgot to add. 

Regarding the bibliography, I could be totally wrong but it could be as simple as adding the path to `_config.yml`. I'm saying this because I can see the complete bibliography when I build the book in my local machine after I add the following lines:
```
bibtex_bibfiles:
    - _bibliography/references.bib
```
I also removed the path from `bibliography.md` - I just left it as:
```
      ```{bibliography}
      ```
```
and it's building fine and showing all bibliography in the book in my local machine but it's not the case in the here when it's deployed in netlify, it says: 
```
WARNING: ../../_bibliography/references.bib not found or not configured in bibtex_bibfiles
```
I could be missing something because it's failing to build here unlike my local machine- I think Sarah and Martina might be able to find it and fix it. Please feel free to revert back to commit `7fada28`- when I added bibtex_bibfiles path in `_config. yml`, if you want to.

**Comment by bsipocz** [comment]
> Yes, both Kristie and @bsipocz recommended that we remove the Travis part completely. It's a mystery to me how it made a comeback in this PR.

I'm afraid that was my mistake at the last rebase I did, as I misinterpreted this conflict `deleted by them: book/website/reproducible-research/ci/ci-travis.md` and skipped your commit that removed it from that PR. 
OTOH, I'm surprised it didn't raise an error during rendering the book, I would have expected an exception for a case like that.

File: book/website/reproducible-research/case-studies/case-study-statistical.md [hunk 1]
```diff
@@ -13,14 +13,15 @@ sections discussed in this {ref}`rr`.
...
-for confirmatory trials.{cite}`Kunzmann2020CS`
+for confirmatory trials{cite:t}`Kunzmann2020CS`.
 - **Authors:** Kevin Kunzmann, Michael J. Grayling, Kim May Lee,
 David S. Robertson, Kaspar Rufibach, James M. S. Wason
 - **Publication month & year**: June 2020
 
 ### Overview
 
-The manuscript{cite}`Kunzmann2020CS` itself is concerned with the problem of
+The manuscript {cite:t
```
Comment (da5nsy) [inline book/website/reproducible-research/case-studies/case-study-statistical.md:23]: flag

**Comment by martinagvilas** [comment]
@malvikasharan @BatoolMM I've made some changes and the bibliography is correctly seen in the netlify deployment now. 

I don't know if the style of the citations is the one you have a preference for, though. In my very personal opinion there are some better styles than the current one that uses abbreviations :) 

Also, I removed a bibliography section from the file `project-design/filenaming.md` because it was inconsistent with other sections where we don't provide the references at the very end, but in the afterword.

Let me know what you think :)

**Comment by malvikasharan** [comment]
@martinagvilas Thank you for fixing the issue. 🙌

I agree that the citation style can be changed as they don't render very well, for example: 
image

I can test a few different styles and see what works the best.

**Comment by malvikasharan** [comment]
@martinagvilas I have replaced `cite:t` with `cite:ps`. It is not as expected or described in Jupyter Book (our bib style is slightly different), but does the job as previously.

**Comment by martinagvilas** [comment]
@malvikasharan looks good to me!

**Comment by malvikasharan** [comment]
Great! Thank you so much for the troubleshooting everyone. Whenever you have time, can one of you approve and merge this?

**Review by BatoolMM — APPROVED** [review]
It looks great to me 🎉

**Pull request merged**